packages/components/src/unit-control/styles/unit-control-styles.js

```diff
@@ -54,14 +54,16 @@ export const ValueInput = styled( NumberControl )`
 const unitSizeStyles = ( { size } ) => {
 	const sizes = {
 		default: {
-			top: 1,
 			height: 28,
+			lineHeight: '24px',
```

Comment:
Adjustments to label height. Abstracted from:
https://github.com/WordPress/gutenberg/pull/23006